<ui version="4.0" >
 <class>SimpleEntryWidget</class>
 <widget class="QWidget" name="SimpleEntryWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>376</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QFrame" name="buttonFrame" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QToolButton" name="breveNoteButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="wholeNoteButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="halfNoteButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QToolButton" name="quarterNoteButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QToolButton" name="note8Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QToolButton" name="note16Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QToolButton" name="note32Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QToolButton" name="note64Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QToolButton" name="note128Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QToolButton" name="breveRestButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QToolButton" name="wholeRestButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QToolButton" name="halfRestButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QToolButton" name="quarterRestButton" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4" >
         <widget class="QToolButton" name="rest8Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QToolButton" name="rest16Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QToolButton" name="rest32Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QToolButton" name="rest64Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QToolButton" name="rest128Button" >
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="voiceList" >
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text" >
       <string>Voice 1</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Voice 2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Voice 3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Voice 4</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QToolButton" name="doubleFlatButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="flatButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="naturalButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QToolButton" name="sharpButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QToolButton" name="doubleSharpButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QToolButton" name="dotButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QToolButton" name="tiedNoteButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QToolButton" name="eraserButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
<!--        <item row="1" column="3" >
         <widget class="QToolButton" name="selectButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item> -->
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="addBarsButton" >
       <property name="text" >
        <string>Add measures</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="importButton" >
       <property name="text" >
        <string>Import</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportButton" >
       <property name="text" >
        <string>Export</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>182</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
